src/liftwing_api/models/revertrisk.py

```diff
@@ -0,0 +1,44 @@
+import requests
+import json
+from liftwing_model import LiftwingModel
+
+class RevertRiskAPIModel(LiftwingModel):
+    def __init__(self, base_url="https://api.wikimedia.org/service/lw/inference/v1/models/{language}-reverted:predict"):
+        super().__init__(base_url)
+        # base url is super because every class that inherits this from the base model will be using it 
+
+    def request_to_revertRiskAPI(self, language: str, revision_id: int):
```

Comment:
the function name needs to be the same as the parent class so it will be `request` and have payload and headers as input.